config/locales/fr.yml

```diff
@@ -158,138 +459,254 @@ fr:
       browse: Parcourir
       documentation: Documentation
       help: Aide
-      landscape: Panorama
+      landscape: Paysage
       login: Connexion
       logout: Déconnexion
-      mappings: Alignements
+      mappings: Mappages
       publications: Publications
-      recommender: Recommendeur
       release_notes: Notes de version
       search_prompt: Rechercher dans %{portal_name} ...
-      submit_feedback: Envoyer un commentaire
-      support: Assistance
+      submit_feedback: Envoyer un retour
+      support: Support
       cite_us: Citez-nous
     footer:
       products: Produits
       ontoportal: OntoPortal
       release_notes: Notes de version
       api: API
       sparql: SPARQL
-      support: Assistance
-      contact_us: Contactez nous
+      support: support
+      contact_us: Contactez-nous
       wiki: Wiki
       documentation: Documentation
-      agro_documentation: Documentation de AgroPortal
-      agreements: Juridique
-      terms: Termes et Conditions
+      agro_documentation: Documentation d'AgroPortal
+      agreements: Légal
+      terms: Termes et conditions
       privacy_policy: Politique de confidentialité
       cite_us: Citez-nous
       acknowledgments: Remerciements
       about: À propos
       about_us: À propos de nous
       projects: D2KAB
       team: Équipe
- 
+    notices:
+      slice: Tranche
+      at: à
+      visit_the_site: visitez le site complet
+
   login:
-    enter_email: Saisissez votre nom d'utilisateur
-    enter_password: Saisissez votre mot de passe
+    invalid_account_combination: "Combinaison nom de compte/mot de passe invalide"
+    authentication_failed: "L'authentification %{provider} a échoué"
+    admin_logged_out: Déconnecté <b>%{old_user}</b>, retourné à <b>%{user}</b>
+    user_logged_out: Vous vous êtes déconnecté avec succès
+    try_again_notice: ". Veuillez réessayer."
+    reset_password_again: ". Veuillez réinitialiser votre mot de passe à nouveau."
+    custom_ontology_set: "L'affichage est maintenant basé sur votre <a href='/account#custom_ontology_set'>Ensemble d'Ontologies Personnalisé</a>."
```

Comment:
We need to keep html elements in the .haml file to keep code modification easier